bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/GC.java

```diff
@@ -133,6 +134,13 @@ public final class GC extends Resource {
  */
 public GC(Drawable drawable) {
 	this(drawable, SWT.NONE);
+	if (drawable instanceof Control) {
```

Comment:
Why is it okay to make the initialization of the `(native)DeviceZoom` values only in one of the `GC` constructors, but not in the other?